agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/quickpulse/QuickPulseDataCollector.java

```diff
@@ -172,6 +173,7 @@ public synchronized void enable(TelemetryClient telemetryClient) {
     counters.set(new Counters());
   }
 
+  @Nullable
```

Comment:
this is mainly used for code analyzers? without it will spotlessApply trigger a warning?
return object type is nullable by default.

Reply:
I updated errorprone in one of the commits, and that required `@Nullable` on return types for methods that return explicit `null`. Eventually we will want to use `@Nullable` consistently everywhere. Conventionally the default is non-null, and only apply `@Nullable` on parameters, fields, method return values that are nullable.